pyproject.toml

```diff
@@ -61,22 +60,29 @@ doc = [
   "numpy",
   "xarray",
   "sphinx-copybutton",
-  "sphinx-design",
   "sphinx-togglebutton",
   # Install nbsphinx in case we want to test it locally even though we can't load
   # it at the same time as MyST-NB.
   "nbsphinx",
   "ipyleaflet",
 ]
+doc = [
```

Comment:
we are now missing `rich` in the documentation build on RDT

Reply:
yep I saw that but I don't know why. `[doc]` inherits from `[doc_base]` so `rich` ought to be installed.

Reply:
when the pyproject.toml file specifies `pydata-sphinx-theme[doc_base]` is it equivalent to `.[doc-base]` or is it looking to the latest available on pipy ?

Reply:
ahh, good point. I think @nicoa made this point in another issue and I still forgot 🤦🏻

Reply:
sorry that was an open question, I actually don't know what is the syntaxe to use to nest extra_require

Reply:
I tried with `.[whatever]` and it fails. If I understand https://github.com/pypa/pip/issues/10393#issuecomment-941885429 correctly then it actually should work (with pip >= 21.2) to use `pydata-sphinx-theme[whatever]` and it will look within the local `pyproject.toml` file (instead of looking at only released versions on PyPI), so I don't know why it's not working like that here.

Reply:
yes, addressed this somewhere (that we first need a (pre-)release) - thanks for pointing to the pip issue, that is a good hint. Unfortunate that it didn't work out still :-(